src/aiokatcp/server.py

```diff
@@ -99,7 +101,7 @@ def sensor_update(self, s: sensor.Sensor, reading: sensor.Reading) -> None:
         msg = core.Message.inform(
             "sensor-status", reading.timestamp, 1, s.name, reading.status, reading.value
         )
-        self.write_message(msg)
+        self.owner._write_async_message(self, msg)
```

Comment:
I thought this sounded familiar. I once had to `server.mass_inform`, I see it uses this under the hood.